buildAll.proj

```diff
@@ -31,7 +41,10 @@
                       src\Microsoft.ServiceFabric.Actors.Wcf\packages.config;
                       src\Microsoft.ServiceFabric.Services\packages.config;
                       src\Microsoft.ServiceFabric.Services.Remoting\packages.config;
-                      src\Microsoft.ServiceFabric.Services.Wcf\packages.config"/>
+                      src\Microsoft.ServiceFabric.Services.Wcf\packages.config;
+                      src\test\unittests\Microsoft.ServiceFabric.Actors.Tests\packages.config;
```

Comment:
Would it be better to restore nugets for tests when only tests are being built, just the product code build can be a little faster if number of nuget packages used by tests increase